website/content/docs/api-gateway/install.mdx

```diff
@@ -37,47 +19,46 @@ Ensure that the environment you are deploying Consul API Gateway in meets the re
             name: consul
           connectInject:
             enabled: true
-          apiGateway:
-            enabled: true
-            image: hashicorp/consul-api-gateway:$VERSION
         EOF
       ```
     </CodeBlockConfig>
 
-1. Install Consul API Gateway using the standard Consul Helm chart or Consul K8s CLI specify the custom values file. Available versions of the [Consul Helm chart](https://github.com/hashicorp/consul-k8s/releases) can be found in GitHub releases.
+The Consul Helm chart deploys the API gateway using the configuration specified in the `values.yaml` file. Refer to [Helm Chart Configuration - `apiGateway`](/consul/docs/k8s/helm#apigateway) for information about the Helm chart configuration options.
```

Comment:
This has changed with the new release. The `apiGateway` stanza is now under the `connectInject` key. 

The existing stanza will remain for the duration of 1.16-1.17 to support users who are transitioning from the legacy to the native implementation.

So the link here should point to the `connectInject.apiGateway` stanza. Even though those docs have not been generated from the Helm values file yet.